zerver/decorator.py

```diff
@@ -29,7 +29,7 @@
 from zerver.lib.mandrill_client import get_mandrill_client
 from six.moves import zip, urllib
 
-from six import text_type
+from typing import Text
 from typing import Union, Any, Callable, Sequence, Dict, Optional, TypeVar
```

Comment:
You should put the `Text` onto the next line rather than making a new one; can you fix this in the various files?